src/lib/__snapshots__/create-config.test.ts.snap

```diff
@@ -100,6 +100,7 @@ exports[`should create default config 1`] = `
       },
       "migrationLock": true,
       "multipleRoles": false,
+      "newTransactional": false,
```

Comment:
I like to be precise in flag names. maybe transactionalDecorator?

Reply:
absolutely!

Reply:
[f61c572](https://github.com/Unleash/unleash/pull/4916/commits/f61c5722c87dfc2bafc741be9ce994e352c13aed)